src/actions/index.js

```diff
@@ -249,6 +250,14 @@ function importProjectFromGist(projectKey, gistData) {
   };
 }
 
+function parsePopcodeJson(files) {
+  const popcodeJsonFile = find(files, {filename: 'popcode.json'});
+  if (!popcodeJsonFile) {
+    return {};
+  }
+  return JSON.parse(get(popcodeJsonFile, 'content', '{}'));
```

Comment:
I think this is safe from XSS / people trying to inject bad stuff into JSON since we don't render this anywhere and the libraries are whitelisted, but please try to think of any vectors of attack I haven't thought of.